src/stringify.ts

```diff
@@ -75,7 +75,7 @@ const BSON_TO_JS_STRING = {
     }
     // The `Binary.buffer.toString` type says it doesn't accept
     // arguments. However it does, and a test will fail without it.
-    return `BinData(${subType.toString(16)}, '${v.toString('base64')}')`;
+    return `BinData(${subType.toString(10)}, '${v.toString('base64')}')`;
```

Comment:
By the way, we’re standardizing on a format that is finally fully compatible with the Node.js driver:

```suggestion
    return `Binary.createFromBase64('${v.toString('base64')}', ${subType.toString(10)})`;
```